examples/grafana/grafana.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     "openshift.io/display-name": Grafana
     description: |
       Grafana server with patched Prometheus datasource.
-    iconClass: icon-cogs
+    iconClass: fa fa-cogs
```

Comment:
the space is intentional?

Reply:
Yeah, it needs to classnames: `fa` and `fa-cogs`

https://fontawesome.com/v4.7.0/icon/cogs/